app/src/main/java/com/android/unio/MainActivity.kt

```diff
@@ -33,11 +32,18 @@ fun UnioApp() {
   // start destination should be Route.AUTH, but for now, we let it be Route.HOME for testing
   // purposes
   NavHost(navController = navController, startDestination = Route.HOME) {
-    navigation(startDestination = Screen.AUTH, route = Route.AUTH) {
-      composable(Screen.AUTH) { LoginScreen() }
-      navigation(startDestination = Screen.HOME, route = Route.HOME) {
-        composable(Screen.HOME) { HomeScreen(navigationActions) }
-      }
+    //    navigation(startDestination = Screen.AUTH, route = Route.AUTH) {
+    //      composable(Screen.AUTH) { LoginScreen() }
+    //      navigation(startDestination = Screen.HOME, route = Route.HOME) {
+    //        composable(Screen.HOME) { HomeScreen(navigationActions) }
+    //      }
+    //      navigation(startDestination = Screen.EXPLORE, route = Route.EXPLORE) {
+    //        composable(Screen.EXPLORE) { ExploreScreen(navigationActions) }
+    //      }
+    //  }
+    navigation(startDestination = Screen.HOME, route = Route.HOME) {
+      composable(Screen.HOME) { HomeScreen(navigationActions) }
+
```

Comment:
Implement here the different screens `Account` and `Saved`

Reply:
I just worked on this and it should be fixed by now.